.github/workflows/update-changelog.yml

```diff
@@ -0,0 +1,20 @@
+name: 'update-changelog'
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+      - 'releases/*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: update-changelog
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+          npm ci
+      - run: |
+          npm run all
```

Comment:
Just style: shouldn't these 2 commands be either
```
- run: |
          npm ci
          npm run all
```
or
```
- run: npm ci
- run: npm run all
```
Since you're not naming the steps I think I prefer option 1